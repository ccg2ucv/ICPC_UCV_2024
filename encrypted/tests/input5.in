836
NZY_P^_j`Naj=;=?