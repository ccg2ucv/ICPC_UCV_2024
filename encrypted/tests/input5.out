CONTEST_UCV_2024